src/main/resources/userEmailTemplate-userRegisterForCourse.html

```diff
@@ -0,0 +1,14 @@
+<p>Hello, ${userName}</p>
+
+<p>Thank you for registering your Google ID with TEAMMATES!</p>
```

Comment:
Let's merge these two sentences as follows:

`Thank you for registering as an instructor in the course: <strong>[${courseId}] ${courseName}</strong> with the Google ID: <strong>${googleId}</strong>.</p>`